### PR TITLE
Move card once tweeted

### PR DESCRIPTION
**Pull request by MichaelKohler**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -116,6 +121,15 @@ async function doStuff() {
             issue_number: issue.number,
             state: 'closed',
         });
+        const availableColumns = await octokit.rest.projects.listColumns({
+            project_id: getProjectIdFromUrl(github.context.payload.project_card.project_url),
```
Comment (MichaelKohler) [inline index.js:125]: This is kinda disappointing, but didn't find a direct link 😢

File: action.yml [hunk 2]
```diff
@@ -16,6 +16,9 @@ inputs:
   column:
     description: 'Column where issues ready to tweet are stored'
     default: 'To Tweet'
+  doneColumn:
+    description: "Column where tweeted issues should be stored"
+    default: 'Tweeted'
```
Comment (MichaelKohler) [inline action.yml:21]: Looking forward to the merge conflict...

File: index.js (hunk 1 above)
Comment (freaktechnik) [reply index.js:125]: This is a REST API, so you're supposed to get that `project_url` to get the project id... From what I can tell octokit only offers https://octokit.github.io/rest.js/v18#custom-requests for that.

File: index.js [hunk 3]
```diff
@@ -43,6 +43,11 @@ function getTweetIdFromUrl(url) {
     return match?.[1] ?? null;
 }
 
+function getProjectIdFromUrl(url) {
+    const match = /^https?:\/\/api\.github.com\/projects\/([0-9]+)\/?$/.exec(url);
```
Comment (freaktechnik) [inline index.js:47]: I'd honestly just split on / and use the last segment...

**Review by freaktechnik — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (freaktechnik) [reply index.js:110]: Just remembered where the similar logic in the monolith was: https://github.com/mozillach/gh-projects-content-queue/blob/master/lib/column.js#L31

**Pull request merged**